### PR TITLE
Fix invalid time sheet link

### PR DESCRIPTION
**Pull request by oliverguenther**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://oliverguenther-fix-time-sheet-incorrect-link.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://oliverguenther-fix-time-sheet-incorrect-link.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: config/initializers/menus.rb [hunk 1]
```diff
@@ -44,7 +44,7 @@
             { controller: '/news', project_id: nil, action: 'index' },
             if: Proc.new { User.current.allowed_to?(:view_news, nil, global: true) }
   menu.push :time_sheet,
-            { controller: '/time_entries', project_id: nil, action: 'show' },
+            { controller: '/time_entries', project_id: nil, action: 'index' },
```
Comment (NobodysNightmare) [inline config/initializers/menus.rb:47]: I talked to @ulferts and @floriank whether this is something we want to have a test for. Something along the lines: When I click the link, I should see a page.

They could not agree whether it should be RSpec or a Cucumber test, but both agreed that there should be a test of some kind.

**tl; dr:** Write a spec for that, decide upon cuke vs rspec yourself ^^

**Comment by oliverguenther** [comment]
@NobodysNightmare Updated with specs. Feel free to merge.

However, the specs do not explicitly test when the permissions (`:view_work_packages`,  `:view_time_entries`, etc.) apply for a default user. It only tests that the buttons show up If these permission are given. The exact detail of what permissions are granted depend on the configuration (and are different from community to test, for example).

If you think this should be the duty of this particular spec, I'll have to extend the tests further.

**Comment by NobodysNightmare** [comment]
@oliverguenther Given the fact that this is such a small PR the specs should be good enough ;)
